bom/pom.xml

```diff
@@ -459,6 +519,11 @@
                 <artifactId>cxf-codegen-plugin</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
```

Comment:
AFAIK plugins are not imported as part of BOMs (please correct me if I am wrong), should not be there

Reply:
Removed maven-plugins